content/english/articles/effective-gnops/onboarding-testnet-validator/index.md

```diff
@@ -2,10 +2,84 @@
 title: Onboarding a testnet validator
 publishDate: 2025-03-11T08:00:00-01:00
 translationKey: "bootnodes-persistent-peers"
-tags: [ "validator", "testnet", "onboarding" ]
+tags: ["validator", "valopers", "testnet", "onboarding"]
 level: Intermediate
 author: None
 summary: "This give a set of information on onboarding a validator node: how to provision the right server and connect it to the testnet"
 ---
 
-## TBD
+# Setting Up a Validator Node in gno.land
+
+## Overview
+
+Validators on **gno.land** are expected to demonstrate their technical expertise and alignment with the project by making continuous and meaningful contributions. **gno.land** abstracts validator management into the `r/sys/vals` realm as a form of a smart contract for modularity.
+
+This guide walks you through the process of registering your validator node into the validator set with a smart-contract. It assumes that you already have an operational validator node running on the testnet. If not, follow [this guide](https://docs.gno.land/gno-infrastructure/validators/validators-running-a-validator) (especially the section "Connect to an Existing Gno Chain").
+
+## Step 1: Creating a `Valoper` object
+
+To be added to the validator set, you must create a **Valoper** object and register it in the `r/gnoland/valopers` realm using the `Register` function. This object allows you to demonstrate to **GovDAO members** why you should be accepted as a validator.
+
+### Information to Include in the `Valoper` object
+
+Ensure to follow the instruction inside the valopers realm. This is an example of details you should provide in your description:
```

Comment:
```suggestion
Be sure to follow the instructions inside the valopers realm. This is an example of the details you should provide in your description:
```

Reply:
fixed in https://github.com/gnoverse/gnops/pull/27/commits/6de91741a5b04b82d09bb30444378d62cb296d94